core/domain/feedback_services.py

```diff
@@ -172,38 +244,58 @@ def get_next_page_of_all_feedback_messages(
     return (result_messages, new_urlsafe_start_cursor, more)
 
 def get_thread_analytics_multi(exploration_ids):
-    """Returns a dict with feedback thread analytics for the given exploration
-    ids.
+    """Fetch all FeedbackAnalytics for all the given exploration ids
+
+    Args:
+        exploration_ids: list of str. A list of exploration id
+
+    Returns:
+        list of FeedbackAnalytics
     """
-    return feedback_jobs_continuous.FeedbackAnalyticsAggregator.get_thread_analytics_multi( # pylint: disable=line-too-long
-        exploration_ids)
+    return feedback_jobs_continuous.FeedbackAnalyticsAggregator.\
+        get_thread_analytics_multi(exploration_ids)
 
 
 def get_thread_analytics(exploration_id):
-    """Returns a dict with feedback thread analytics for the given exploration.
+    """Fetch all FeedbackAnalytics for the given exploration id
 
-    The returned dict has two keys:
-    - 'num_open_threads': the number of open feedback threads for this
-        exploration.
-    - 'num_total_threads': the total number of feedback threads for this
-        exploration.
+    Args:
+        exploration_id: str.
+
+    Returns:
+        list of FeedbackAnalytics
     """
-    return feedback_jobs_continuous.FeedbackAnalyticsAggregator.get_thread_analytics( # pylint: disable=line-too-long
-        exploration_id)
+    return feedback_jobs_continuous.FeedbackAnalyticsAggregator.\
+        get_thread_analytics(exploration_id)
 
 
 def get_total_open_threads(feedback_thread_analytics):
-    """Returns the count of all open threads for the given
-    FeedbackThreadAnalytics domain objects."""
+    """Get the count of all open threads for the given FeedbackThreadAnalytics
+
+    Args:
+        feedback_thread_analytics: FeedbackThreadAnalytics
+
+    Returns:
+        int. The count of all open threads for the given FeedbackThreadAnalytics
+    """
     return sum(
         feedback.num_open_threads for feedback in feedback_thread_analytics)
 
 
 def create_suggestion(exploration_id, author_id, exploration_version,
                       state_name, description, suggestion_content):
-    """Creates a new SuggestionModel object and the corresponding
-    FeedbackThreadModel object."""
+    """Create a new SuggestionModel and the corresponding FeedbackThreadModel
 
+    Args:
+        exploration_id: str. The exploration id the suggestion belongs.
+        author_id: str. ID of the user who submitted the suggestion.
+        exploration_version: int or None. The exploration version for
```

Comment:
I think @wxyxinyu means, what does it mean conceptually? Docstrings need to explain this, i.e. `If None, this indicates that ...`.

Looking at this, I'm not actually sure why None is a possibility. Can you find an instance in the codebase that calls this with None?